src/pages/workspace/WorkspaceInvitePage.js

```diff
@@ -242,12 +242,14 @@ class WorkspaceInvitePage extends React.Component {
             Boolean(this.state.userToInvite),
             this.state.searchValue,
         );
+        const policyName = lodashGet(this.props.policy, 'name');
```

Comment:
I know this is pedantic, but could you put a new line below this to match the other ones?